src/assets/icon-button.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+
+interface IconButtonProps {
+  icon: string;
+  key: string;
+  className: string;
+  onClickButton: () => void;
+  url?: string;
+}
+
+export const IconButton = (props: IconButtonProps) => {
+  const styleButton = {
+    fontSize: "100%",
+    fontFamily: "inherit",
+    border: 0,
+    padding: 0
+  };
```

Comment:
Since these styles are unchanging they can be in the CSS. Also, it's not clear to me why we need to say anything about the font at all.

Reply:
I've moved the styles to the CSS without the font related styles (I was thinking of overall button overrides as buttons usually include some amount of text when I wrote this).